upi/metal/terraform.tfvars.example

```diff
@@ -42,6 +42,16 @@ matchbox_trusted_ca_cert = "matchbox/tls/ca.crt"
 // The Project ID for Packet.net where servers will be deployed.
 packet_project_id = ""
 
+// The Packet.Net device plan slug.
+packet_plan = "c1.small.x86"
+
+// The Packet.Net facilities code(s) to be used.
+packet_facility = "any"
+
+// The UUID of the hardware reservation where you want this device deployed on Packet.net.
+// The value "next-available" can also be used instead of a specific UUID. 
+packet_hardware_reservation_id = "next-available"
```

Comment:
This should be "" to preserve the current default behavior of not using reserved instances.

Reply:
Fixed.